tests/test_core.py

```diff
@@ -5650,8 +5653,10 @@ def test_unistd_unlink(self):
       if fs == 'WASMFS':
```

Comment:
I believe this is now dead code. Relatedly, there is a lot of setup logic here that is not directly related to unlinking. Is that logic going to be repeated for multiple tests? It might be worth abstracting away if so.

Reply:
Good catch, removed the dead code now.

I'm not sure if this setup code repeats in other tests. If I see it again in another test that I get to later I'll keep a lookup for refactoring opportunities.